src/assets/styles.scss

```diff
@@ -154,6 +153,25 @@ hr {
     margin: 0.6em 0em;
 }
 
+/* Index.vue page B-card styling */
```

Comment:
Here you should put first the container and put the other classes inside (as the footer for example)
Then you are sure that if similar class names are used somewhere else then it is not modified

Reply:
Always start by the highest level and then add the other levels inside

Reply:
I'll do that now. Thank you so much!